src/components/Button/Button.tsx

```diff
@@ -26,6 +27,7 @@ import {
 } from '../../styles/style-helpers';
 import { ReturnType } from '../../types/return-type';
 import { useComponents } from '../ComponentsContext';
+import useClickTracker from '../../hooks/use-click-tracker';
```

Comment:
So, in collector we are it triggers, did we rename to avoid confusion?

Reply:
How about `useTrackingHandler`? or any suggestion? I'm running out of ideas.

Reply:
I like `useClickHandler`.